torch/onnx/symbolic_helper.py

```diff
@@ -826,6 +867,28 @@ def _handle_reduce_dim_none(g, self, op_name):
         return g.op(op_name, self, keepdims_i=1)
     return g.op(op_name, self, keepdims_i=0)
 
+def _dequantize_helper(g, qtensor, qdtype=None):
```

Comment:
Not super confident about the conventions, but I think we should not name it with a leading underscore if it's meant to be used outside this file? Or what is the leading underscore supposed to mean in this context?

Reply:
no idea .. removing underscore.